packages/common/driver-definitions/src/driverError.ts

```diff
@@ -145,6 +145,11 @@ export interface IDriverErrorBase {
 	 * See OnlineStatus enum in driver-utils package for expected values.
 	 */
 	online?: string;
+
+	/**
+	 * Whether service was reachable and we got some response from service.
+	 */
+	endpointReachable?: boolean;
```

Comment:
I think it should be "Reached" not "Reachable".
I.e. we know (based on this property) that we reached or not service end-point in the past.